cmd/system-probe/config/adjust.go

```diff
@@ -37,7 +37,7 @@ func Adjust(cfg config.Config) {
 		// enable the connections/network check.
 		log.Warn(deprecationMessage(spNS("enabled"), netNS("enabled")))
 		// ensure others can key off of this single config value for NPM status
-		cfg.Set(netNS("enabled"), true)
+		cfg.SetForSource(netNS("enabled"), true, config.SourceAgentRuntime)
```

Comment:
I don't think `config.SourceAgentRuntime` applies to anything in this file. These are mostly defaults and deprecations. Nothing is coming from the user.

Reply:
SourceAgentRuntime corresponds to settings that the agent set itself
What the user sets is SourceCLI